spec.csv

```diff
@@ -0,0 +1,39 @@
+integration,metric_name,metric_type,metric_enabled,metric_description
+RabbitMQ,node.fileDescriptorsTotalUsed,Gauge,true,Count of File Descriptors Used
+RabbitMQ,node.diskSpaceFreeInBytes,Gauge,true,Current free disk space
+RabbitMQ,node.totalMemoryUsedInBytes,Gauge,true,Memory used in bytes
+RabbitMQ,node.averageErlangProcessesWaiting,Gauge,true,Average number of Erlang processes waiting to run
+RabbitMQ,node.fileDescriptorsUsedSockets,Gauge,true,Number of file descriptors used as sockets
+RabbitMQ,node.partitionsSeen,Gauge,true,Number of network partitions seen per node
+RabbitMQ,node.running,Gauge,true,Node Running 
+RabbitMQ,node.hostMemoryAlarm,Gauge,true,Host Memory Alarm
+RabbitMQ,node.diskAlarm,Gauge,true,Node Disk Alarm
+RabbitMQ,vhost.aliveness,Gauge,true,"Check of aliveness of a Vhost. 0 = OK, 1 = Failure"
+RabbitMQ,vhost.currentConnections,Gauge,true,Number of current connections to a given rabbitmq vhost
+RabbitMQ,vhost.connectionsSpecifiedState,Gauge,true,Number of connections in the specified connection state
+RabbitMQ,exchange.bindings,Gauge,true,Number of bindings for a specific exchange
```

Comment:
Probably be more convenient if this exchange metric was moved down with the other exchange metrics.